modules/network-observability-nodes-taints-tolerations.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// network_observability/network-observability-scheduling-resources.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-multi-tenancy{context}"]
+= Using tolerations to control Network Observability deployment in specific nodes
```

Comment:
This module is a "concept" but the title starts with a gerund. The I[BM quick reference](https://spaces.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=Titles+and+headings) suggests using a noun for concepts. Perhaps it can be written as follows:
```suggestion
= Network Observability deployment in specific nodes
```